### ISSUE TITLE
随便记记

### ISSUE THREAD

**Issue by suukii** [suukii/daily#8]
- Use `chrome.contextMenus` API to add items to the Google Chrome context menu.
- Use `window.getSelection().toString()` API to get selected text on the page.

**Comment by suukii** [comment]
# How to get the auto-generated key of Chrome extensions

1. [Package your extension to get a .crx file](http://www.dre.vanderbilt.edu/~schmidt/android/android-4.0/external/chromium/chrome/common/extensions/docs/packaging.html)
2. Load the `.crx` file into `chrome://extensions` under development mode.
3. Copy the `id` of your extension shown in `chrome://extesions`.
4. `cat ~/Library/Application\ Support/Google/Chrome/Default/Extensions/_<extensionId>_/_<versionString>_/manifest.json | grep key`

**ps**

`load unpacked` will not do. The ID generated for an unpacked extension is just a temporary hash.

**Comment by suukii** [comment]
# i18n

https://github.com/projectfluent/fluent.js

**Comment by suukii** [comment]
# Error: Node Sass version 5.0.0 is incompatible with ^4.0.0

**TL;DR**

install the 4.14.1 version: `yarn add node-sass@4.14.1`

https://stackoverflow.com/questions/64625050/error-node-sass-version-5-0-0-is-incompatible-with-4-0-0
